test/python/test_qi.py

```diff
@@ -207,14 +209,6 @@ def test_entanglement_of_formation_invalid_input(self):
         self.assertEqual(fake_stout.getvalue().strip(), expected)
         self.assertIsNone(res)
 
-    def test__eof_qubit(self):
```

Comment:
This text makes no sense now?

Reply:
It's internal function of the main function which is already tested